extras/csv-examples/motions-demo_de.csv

```diff
@@ -1,4 +1,4 @@
-"Bezeichner";"Titel";"Text";"Begründung";"Antragsteller";"Sachgebiet"
+identifier;title;text;reason;submitter;category
```

Comment:
English strings are required in csv header because these strings are used as object properties (e.g. motion.identifier - If there is no column header name with "identifier" the field cannot be imported.)